<launch>
	<arg name="odom" default="/base_pose_ground_truth"/> 
	<arg name="lidar" default="/base_scan"/> 
	<arg name="vel" default="/cmd_vel"/> 
	<arg name="position_x" default="-3.0"/> 
	<arg name="position_y" default="-1.0"/> 
	<arg name="target_x" default="0.0"/> 
	<arg name="target_y" default="0.0"/> 
	<arg name="kp" default="0.5"/> 
	<arg name="kd" default="0.1"/> 


	<node name="stageros" pkg="stage_ros" type="stageros" args="$(find controller_PD)/world/create_hokuyo.world"/>
	<node pkg="controller_PD" type="diferencial_PD.py" name="controller_PD" output="screen"> 
		<param name="odom"  value="$(arg odom)"/>
		<param name="lidar"  value="$(arg lidar)"/>
		<param name="vel"  value="$(arg vel)"/>
		<param name="position_x"  value="$(arg position_x)"/>
		<param name="position_y"  value="$(arg position_y)"/>
		<param name="target_x"  value="$(arg target_x)"/>
		<param name="target_y"  value="$(arg target_y)"/>
		<param name="kp"  value="$(arg kp)"/>
		<param name="kd"  value="$(arg kd)"/>
	</node>
</launch>
